sdk/appconfiguration/azure-spring-cloud-appconfiguration-config/src/main/java/com/azure/spring/cloud/config/feature/management/entity/DynamicFeature.java

```diff
@@ -0,0 +1,82 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.spring.cloud.config.feature.management.entity;
+
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Defines the assigner and variants of a Dynamic Feature
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class DynamicFeature {
+
+    @NotBlank
+    @JsonProperty("id")
+    private String name;
+
+    @NotBlank
+    @JsonProperty("assigner")
```

Comment:
Do we need this @JsonProperty?

Reply:
I mean this `JsonProperty("assigner")` equals to the field name `assigner`, so this can be skipped?